src/NuGet.Clients/NuGet.PackageManagement.UI/Utility/ReadMeMarkdownToolWindow.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.Shell;
+using System.Runtime.InteropServices;
+using System.ComponentModel.Design;
+using Microsoft.VisualStudio.Shell.Interop;
+using System.Windows.Controls;
+
+namespace NuGet.PackageManagement.UI
+{
+    [Guid(Constants.ReadMeMarkdownToolWindowGuid)]
+    public class ReadMeMarkdownToolWindow : ToolWindowPane
```

Comment:
You seem to have two files that have a class with the same name. This one and src/NuGet.Clients/NuGet.PackageManagement.UI/ReadMeMarkdownToolWindow.cs. Remove the extra one.